docs/r2v/data_processing_quick_start.md

```diff
@@ -141,6 +163,78 @@ objects in the cloud become more apparent:
 Once the values have been determined, place them in the 
 `config/sensor_offset.json` file for use in the pipeline.
 
+## Calibrating Texture Mapping
+
+After the mesh data has been created, the Texture Mapping Alignment Tool can
```

Comment:
Probably want to make this even more clear that the next step "Running the pipeline" needs to at least run through meshing first

Reply:
Updated.